content/projects/linux/_index.md

```diff
@@ -58,26 +64,59 @@ Create an empty file named exercise.txt and move this file to the /tmp direc
 5. change the working directory to the previous one (cd -)
 6. locate pad_copy.txt
 
-Note: foreach instruction write a script and save it under this task
+Note: for each instruction write a script and save it under this task
 
 ## Task 6 Find commands
+1. Write a script to find all files ending with 'pdf' on your computer
+2. Write a second command that takes the result of the previous search and copy  into a folder of your choice. 
+3. Write a command to display files that where modified today.
+
+### Resources 
 
-1. Create a script that searches the home directory for all existing files
-2. change directory to My document, create a file named search_result.text. Save and change directory to Descktop
-3. write a scritpt to retrive all directories,files and subdirectories, write the result to search_result.txt
+1. https://www.geeksforgeeks.org/find-command-in-linux-with-examples/
+2. https://unix.stackexchange.com/questions/70455/how-to-run-find-exec-script
 
-## Resources
 
-1. https://bit.ly/2VQbkDK
 
-## Task 7 Installing softwares
+## Task 7 Text editor
+
+1. Using nano text editor create a file named my_bio.txt
+2. Save the file and close the editor
+3. create a folder named my_files and move my_bio.txt within.
+
+### Resources
+
+1. https://www.lifewire.com/beginners-guide-to-nano-editor-3859002
 
-## Task 8 Linux documentation how to
+# Advanced
 
-## Task 9 File operation
 
-## Task 10 Text editor
+## Task 8 User environment
 
-## Task 11 User environment
+1. Use echo to display Hello followed by your username. (use a bash variable!)
+2. Create a variable myName with a value containing your full name.
+3. Copy the value of $LANG to $MyLANG.
+4. List all current shell variables.
+5. Create a nodejs/python script that fetch all created variable and diplays on the creen for nodejs, in termina for python.
```

Comment:
2 x typos: please change creen to screen and termina to terminal

Reply:
Done